repository/ImmutableObjects.package/ImmutableObjectTest.class/instance/assertPropertiesOf..st

```diff
@@ -0,0 +1,4 @@
+test objects
+assertPropertiesOf: anImmutableObject
+
+	self assert: false.
```

Comment:
This leads to a failing test if `assertPropertiesOf` is called. Is this intended behavior?